requirements.txt

```diff
@@ -16,3 +16,4 @@ sh
 coverage
 pytest-cov
 imageio
+pscript
```

Comment:
?

Reply:
That can go, I've given up on the slider bar to control marker size for now.